plugins/SoundFontPlayer/SoundFontPlayer.cpp

```diff
@@ -121,8 +121,8 @@ struct Sf2PluginData
 
 
 
-Sf2Instrument::Sf2Instrument( InstrumentTrack * _instrument_track ) :
-	Instrument(_instrument_track, &sf2player_plugin_descriptor, nullptr, Flag::IsSingleStreamed),
+SoundFontInstrument::SoundFontInstrument( InstrumentTrack * _instrument_track ) :
+	Instrument(_instrument_track, &soundfontplayer_plugin_descriptor, nullptr, Flag::IsSingleStreamed),
```

Comment:
(Non-blocking) Per the Coding Conventions [re: parentheses](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces), [re: pointers](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers), [re: function parameters](https://github.com/LMMS/lmms/wiki/Coding-Conventions#function-parameters):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

and,

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

and, 

> Some older code prefixed function parameters with an underscore `_`. Function parameters SHOULD NOT be prefixed with an underscore.

```suggestion
SoundFontInstrument::SoundFontInstrument(InstrumentTrack* instrument_track) :
	Instrument(instrument_track, &soundfontplayer_plugin_descriptor, nullptr, Flag::IsSingleStreamed),
```